src/features/commands/report.js

```diff
@@ -47,7 +47,8 @@ module.exports = {
 
     async execute (client, interaction) {
         // disabled for Apple community
-        const isPremium = await db.getServerConfig(interaction.guild.id).then(config => config[0]?.isSubscribed)
+        const isPremium = true;
```

Comment:
Accidental change which shouldn't have been commited.

Reply:
Ah yeah, that was from my botched test environment.

Reply:
🚔

Reply:
Mmm piracy